client/src/containers/UserReviews/index.js

```diff
@@ -19,17 +22,13 @@ class UserPosts extends Component {
     userPosts
```

Comment:
If `userPosts` is array, you should rewrite your check like this: `userPosts.length`
Empty array always will be **true**

Reply:
Yes, I know, thank you! 😄 
The problem is that  `userPosts` is not always an array, sometimes it can be undefined... So this check is to avoid an exception. But after your comment, I got an idea that in case if the list is empty is better to show some message instead of an empty table. 🤔